certified-connectors/Zvanu parvaldnieks/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3491 @@
+          {
+            "name": "initial_record_id",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "default": 0,
+            "x-ms-visibility": "internal",
+            "description": "Initial record ID for querying only newly created records.",
+            "x-ms-summary": "Initial record ID"
+          },
+          {
+            "name": "limit",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "x-ms-visibility": "important",
+            "description": "Limit how many results to retrieve.",
+            "x-ms-summary": "Limit"
+          },
+          {
+            "name": "offset",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "x-ms-visibility": "important",
+            "description": "Used to specify from which result we want the data to retrieve.",
+            "x-ms-summary": "Offset"
+          },
+          {
+            "name": "destination",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The number which call was placed to.",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Destination number"
+          },
+          {
+            "name": "source",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The number of calling party.",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Source number"
+          },
+          {
+            "name": "direction",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "\"in\" or \"out\" to indicate if the call was incoming or outgoing.",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Direction"
+          },
+          {
+            "name": "contact_id",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Unique ID of the Employee involved in the call.",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Contact ID"
+          },
+          {
+            "name": "date_from",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "format": "date-time",
+            "description": "Starting date in the filtered date range.",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Date from"
+          },
+          {
+            "name": "date_to",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "format": "date-time",
+            "description": "End date in the filtered date range.",
+            "x-ms-summary": "Date to"
+          }
+        ],
+        "x-ms-trigger-hint": "Triggers when new call recording is created",

```

Comment:
@einarshamrihs please make sure everything is grammatically correct and free of typographical errors, you can follow [these guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).